### ISSUE TITLE
Svelte, streaming resets yRange

### ISSUE THREAD

**Issue by frbju** [project-nv/night-vision#23]
### Describe the bug

I generate new data and add it to the chart, i expect the chart to reload the data but not reset yRange  and push the data left along xrange so that new candles don't go off chart.

### Reproduction

https://codesandbox.io/s/svelte-streaming-chart-ijd3l5?file=/stockchart.svelte

### Javascript Framework

svelte

### Validations

- [X] Read the [docs](https://nightvision.dev/guide/intro/getting-started.html).
- [X] Check that there isn't [already an issue](https://github.com/project-nv/night-vision/issues) that reports the same bug to avoid creating a duplicate.
- [X] Make sure this is a NightVision issue and not a framework-specific issue.
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by C451** [comment]
Yep, already fixed once, this bug reappeared. But you are still doing it wrong: 
https://codesandbox.io/s/svelte-streaming-chart-forked-3djv18?comment=2b387054-48eb-4758-b40e-97527eb1abaf

You shouldn't reset the whole dataset on every tick

**Comment by frbju** [comment]
This was my first solution
https://codesandbox.io/s/svelte-streaming-chart-ijd3l5?file=/stockchart.svelte

But i didnt like that i need to have something in the initial data for it to work.  I would like to just say data: []
Then update it like above example.

**Comment by frbju** [comment]
For the example 10, i just dont understand how that actually works,, my skills is limited.

**Comment by C451** [comment]
If so, you probably need to wait for the high-level Data API. This is a low-level API, meaning no auto-updates.

**Comment by frbju** [comment]
:) Will build the rest of my program in the meantime. Thanks for all the help, i will try to figure stuff out.

**Comment by frbju** [comment]
Got it working the correct way :) Not sure if its in the loop or not but it would be nice if 
1 ) You as a user can provide an empty data [] on init.
2 ) Each overlay got an easier access ex: chart.find('sma14').data = [ new data ]

**Comment by QuantFreedom1022** [comment]
@frbju  are you still working with this project ... i have seen you are posting some of the same types of questions i am ... so if you are still working with this project i would love to chat with you about how you solved some of your problems
